examples/ios/objc/Draw/Info.plist

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
All other examples share the same `Info.plist`. Is there a reason this example doesn't? This may be the reason why this example fails to validate on certain CI configurations:

```
xcodebuild: error: Unable to find a destination matching the provided destination specifier:
		{ name:iPhone 6 }
```

See Xcode [7.3.1](https://ci.realm.io/job/objc_pr/4262/configuration=Release,swift_version=2.2,target=ios-static/consoleFull) and [8.0](https://ci.realm.io/job/objc_pr/4262/configuration=Release,swift_version=3.0,target=ios-static/consoleFull) failures for more details.